src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  background-position: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
```

Comment:
apply background `shorthand`

Reply:
@Viktor-Kost if i raise background-image: url(./images/star.svg); and no-repeat, the stars generally do not repeat, what is the best way to do it?